src/components/collapse/Collapse.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactNode, useState } from 'react';
+import { IconAngleDown, IconAngleUp } from 'hds-react';
+import './Collapse.scss';
+
+interface BaseCollapseProps {
+  collapseContentId: string;
+  isOpen: boolean;
+}
+
+type ToggleFn = () => void;
+
+interface CollapseProps extends BaseCollapseProps {
+  title: string;
+  children: ReactNode;
+}
+
+interface CollapseButtonProps extends BaseCollapseProps {
+  toggleOpen: ToggleFn;
+  children: ReactNode;
+}
+
+const CollapseButton = ({
+  collapseContentId,
+  isOpen,
+  toggleOpen,
+  children,
+}: CollapseButtonProps): JSX.Element => {
+  const onKeyDown = (event: React.KeyboardEvent<HTMLButtonElement>): void => {
+    if (event.keyCode === 13) {
+      toggleOpen();
+    }
+  };
+
+  return (
+    <button
+      type="button"
+      aria-expanded={isOpen}
+      aria-controls={collapseContentId}
+      className="collapse-button"
+      onKeyDown={onKeyDown}
+      onClick={(): void => toggleOpen()}>
+      {children}
+      {isOpen ? (
+        <IconAngleUp className="collapse-button-icon" aria-hidden="true" />
+      ) : (
+        <IconAngleDown className="collapse-button-icon" aria-hidden="true" />
+      )}
+    </button>
+  );
+};
+
+export default ({
```

Comment:
export default () => or export default function() {}? export default function() syntax is applied in HaukiNavigation component. I don't have a strong opinion on this, but think we should mainly try using only one of the options

Reply:
I can change it to match the previous ones